src/PKSim.Core/Services/ExpressionProfileUpdater.cs

```diff
@@ -112,7 +128,7 @@ public ICommand UpdateMoleculeName(ExpressionProfile expressionProfile, string n
 
       private IOSPSuiteCommand renameMolecule(IndividualMolecule molecule, ISimulationSubject simulationSubject, string newMoleculeName)
       {
-         return _individualExpressionTask.RenameMolecule(molecule, newMoleculeName, simulationSubject.Individual);
+         return new RenameMoleculeInSimulationSubjectCommand(molecule, simulationSubject, newMoleculeName, _executionContext).Run(_executionContext);
```

Comment:
@Yuri05 I hae moved around some of the methods that were in molecule specific stuff but in fact it was really for expression profile